.changeset/calm-suits-return.md

```diff
@@ -0,0 +1,13 @@
+---
+"pie-docs": minor
+"pie-monorepo": minor
+"pie-storybook": minor
+"@justeattakeaway/pie-wrapper-react": patch
+---
+
+[Added] - `turbo.json` for some individual packages (storybook, docs, icons-react, icons-vue and icons-webc)
+[Removed] - generated vue icons from source control
+[Changed] - use latest turbo version
+[Changed] - refactor root turbo config
+[Changed] - update custom-elements.json
```

Comment:
I think this has been removed for version control now, so can likely get rid of this line?